bin/Makefile

```diff
@@ -11,7 +11,7 @@ install :
 	chmod u+w "$(EUPS_DIR)/bin"
 	@ find "$(EUPS_DIR)/bin" -type d -exec chmod u+w {} \;
 	@ find "$(EUPS_DIR)/bin" -name "*.pyc" -exec chmod u+w {} \;
-	@ ./mksetup "$(EUPS_PYTHON)" "$(EUPS_DIR)" "$(EUPS_PATH)" "$(SETUP_ALIASES)"
+	"$(EUPS_PYTHON)" ./mksetup "$(EUPS_PYTHON)" "$(EUPS_DIR)" "$(EUPS_PATH)" "$(SETUP_ALIASES)"
```

Comment:
Was removal of the `@` deliberate?

Reply:
Yes.

Reply:
Actually, this commit can probably be dropped.  I don't understand why `./mksetup` blowing up with a bad shebang line was not fatal.